src/bentoml/_internal/runner/runnable.py

```diff
@@ -153,3 +157,4 @@ class RunnableMethodConfig:
     batch_dim: tuple[int, int]
     input_spec: AnyType | tuple[AnyType, ...] | None = None
     output_spec: AnyType | None = None
+    is_stream: bool = False
```

Comment:
same here, this boolean can become a property and can be calculated based on whether this runnablemethod is an asyncgen, gen or not.

Reply:
Agree to remove it from `Runnable.method()` and `svc.api()`, but having such field in the config is still valuable, to cache and prevent redundant calculation, compared to a property.

Reply:
+1 on caching this for later use, if necessary.